slave-dotnet20/test/run

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
```

Comment:
it's called from the build.sh script when you run make test:
https://github.com/openshift/jenkins/blob/master/hack/build.sh#L66

which means we call it as part of our CI infrastructure.